packages/radio-group/src/LionRadioGroup.js

```diff
@@ -94,7 +94,7 @@ export class LionRadioGroup extends LionFieldset {
 
   __triggerCheckedValueChanged() {
     const value = this.checkedValue;
-    if (value && value !== this.__previousCheckedValue) {
+    if (value != null && value !== this.__previousCheckedValue) {
```

Comment:
Can we change it into: `value !== undefined && value !== null && ...`?

Reply:
I will resolve this, as `value != null` is the proper way to test if a value is not null or undefined.

http://adripofjavascript.com/blog/drips/equals-equals-null-in-javascript.html 
https://github.com/airbnb/javascript/blob/57ff032b0740ba2a46af4bc40cf5638a3e62a365/packages/eslint-config-airbnb-base/rules/best-practices.js#L40